src/network/graphql/queries.js

```diff
@@ -194,8 +195,8 @@ class GraphQuery {
  * @param {object} args Arguments for the query.
  * @return {object} Query result.
  */
-export async function events(client, args) {
-  const res = await new GraphQuery(client, 'events', args).execute();
+export async function events(client = apollo, args) {
```

Comment:
remove change. 

you should pass the `client` instance from `AppStore.graphqlClient`. This is because of MobX does have the right reference/context when I was setting up the subscriptions. If you notice the constructor of `AppStore` I pass in the apollo client instance and store it there. All GraphQL calls should be using this instance now.